brewtils/plugin.py

```diff
@@ -368,6 +379,35 @@ def _hook(exc_type, exc_value, traceback):
 
         sys.excepthook = _hook
 
+    def get_system_dependency(self, require, timeout=300):
+        wait_time = 0.1
+        while timeout > 0:
+            system = self._ez_client.find_unique_system(name=require)
```

Comment:
I have to check, but if we have access to the Garden name of the local plugin, we should pass that forward as a filter criteria.

Reply:
local=True seems to be ok